### DISCUSSION TITLE
Way to Customize Style of "Click to add control" Popover?

### DISCUSSION THREAD

**Discussion by davidbrinton** [optimajet/formengine#43] [Q&A] [answered]
I need to do some styling on the popover that displays when you click the + icon, then click "before" or "after" on an item in document when designing a form.

![image](https://github.com/user-attachments/assets/653c9b72-1d36-4b3e-8e81-ad1fbd6e8d0c)

This is the resulting popover that I need to restyle.

![image](https://github.com/user-attachments/assets/0616f6d5-fe94-4055-b5be-b41ae09fb9bc)

I'm looking at the docs for customizing the designer (https://formengine.io/documentation/designer-customization) but it doesn't mention this window. I would like to be able to add styling or a custom class to it so that I can more easily target the correct HTML. Is there a way in the library to do this?

**Comment by sergeythrees** [comment, marked as answer]
@davidbrinton 
We haven't added the customization option for this window yet, but now we may add it. We will inform you about the decision when we discuss it with colleagues.
